SPECS/perl-Test-Deep/perl-Test-Deep.spec

```diff
@@ -57,12 +57,17 @@ find %{buildroot} -type f -name .packlist -delete
 make test
 
 %files
+%license META.yml
```

Comment:
Let's use the `LICENSE.PTR` file to be consistent with how we do it elsewhere. Discussed offline.